internal/pkg/cache/redis/redis.go

```diff
@@ -46,6 +47,7 @@ func NewRedisWithConfig(cfg *config.RedisConfig) *Redis {
 	}
 
 	c := redis.NewClient(o)
+	c.AddHook(redisotel.NewTracingHook())
```

Comment:
@Piotr-MPG Is it possible to add a name for this so that the function (e.g. get) get prefixed with redis.get and redis.set in the traces?

Reply:
I don't see a way to customise it in the redisotel api. It's not a complex integration so we could write a custom hook if it was deemed worth it.